src/main/java/org/example/CalendarManagement/thriftclients/implementation/Client.java

```diff
@@ -0,0 +1,58 @@
+package org.example.CalendarManagement.thriftclients.implementation;
+
+import org.apache.thrift.TException;
+import org.apache.thrift.protocol.TBinaryProtocol;
+import org.apache.thrift.protocol.TProtocol;
+import org.apache.thrift.transport.TSocket;
+import org.apache.thrift.transport.TTransport;
+import org.example.CalendarManagement.thriftclients.interfaces.ClientInterface;
+import org.example.CalendarThriftConfiguration.MeetingSvc;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+@Component
+@Profile("!test")
+public class Client implements ClientInterface
```

Comment:
here also name looks generic (if we are handling meetings, the class name could be like ThriftMeetingsServiceClientImpl and the interface could be like MeetingsServiceClient